src/components/pages/BeregnPage.tsx

```diff
@@ -8,7 +8,7 @@ import { FormContext } from '@/contexts/context'
 import LoadingComponent from '../LoadingComponent'
 import { produce } from 'immer'
```

Comment:
Jeg lurer på hvor nødvendig `immer` er i en så liten og enkel applikasjon. Det hadde vært interessant og se hvor mange kb den tar i deres bundle og ta en ordentlig vurdering på det. Hvis den brukes kun her for å sørge immutability i deres payload, kan man kanskje like godt kode det med bruk av spread operator?

Dette kan vurderes utenfor denne PR'en her.

Reply:
Jeg kan tagge Martin her hvis han er interessert i å være med i denne vurdering :) @martinstorvoll

Reply:
Kan gå igjennom det på standup, men tanken min her er at immer er i hovedsak brukt for mere lettleselig kode ved å abstrahere bort selve oppdatering av objektet, slik at man i resten av koden heller kan fokusere på akkurat hvilken del av objektet som skal oppdateres. Måten et objekt oppdateres på nå er også type-safe. I tillegg hjelper den med immutability som er en liten bonus. 

Mtp. bundle size er ikke immer stor, og har ingen påvirkning på load times. Jeg mener da at hvor mye lettere og maintainable det er å behandle state er verdt prisen å betale for de kb-ene :)

Reply:
Veldig bra med bevisst valg. Den kjøper jeg 👍